.storybook/preview.tsx

```diff
@@ -232,12 +232,12 @@ const preview: Preview = {
                     {
                         value: "default",
                         icon: "circlehollow",
-                        title: "Wonder Blocks (default)",
+                        title: "Legacy (Classic Wonder Blocks)",
```

Comment:
Aligning the theme labels with the tokens doc Caitlyn is working on. I've let the underlying ids the name! Do we want to update the underlying theme ids too? (This can be a later thing, I can create a ticket for it!)

Reply:
Good point! I missed this in the doc. I asked a question about this: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/3683221530/Color?focusedCommentId=4230512641

I'm not sure if we should call it `Legacy` at this point (maybe too early), but I'd love to hear your thoughts about this (and Caitlyn's as well).